Gemfile.lock

```diff
@@ -19,6 +19,7 @@ GEM
     execjs (2.6.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
+    ffi (1.9.10)
     ffi (1.9.10-x64-mingw32)
```

Comment:
I'm not sure about the addition of ffi. Nor how it relates to the -x64-mingw32 version. Will need to investigate.